data/user_data_base.ps2

```diff
@@ -104,36 +103,18 @@ function Get-HudlRoleAttributes {
         $environmentPrefix = $roleSplit[0]
         $serverRole = $roleSplit[1]
         $group = "monolith"
-        # special case for queuedservices and admin here
-        if ($serverRole -eq "queues" -or $serverRole -eq "admin") {
```

Comment:
I haven't done it for queued services yet, but it'd be done in the cookbook instead like how admin is.